src/plugins/intel_cpu/tests/functional/specific_tests.cmake

```diff
@@ -38,6 +38,7 @@ if(DEFINED ENABLE_CPU_SUBSET_TESTS_PATH)
     ${CMAKE_CURRENT_SOURCE_DIR}/shared_tests_instances/set_device_name.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/test_utils/cpu_test_utils.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/test_utils/fusing_test_utils.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/single_layer_tests/classes/eltwise.cpp
```

Comment:
Why we need this?

Reply:
Removed. If `ENABLE_CPU_SUBSET_TESTS_PATH` should include tests instantiations only then we should include test implementations in cmake. If not, then we should include tests instantiation and implementation. Any way, it was removed.